EDMC.py

```diff
@@ -99,14 +97,16 @@ def main():
                 print(f'{appversion} ({newversion.title!r} is available)')
             else:
                 print(appversion)
-            sys.exit(EXIT_SUCCESS)
+            return
```

Comment:
```suggestion

            return
```
newline on scope change